tests/test_api.py

```diff
@@ -0,0 +1,119 @@
+
+###############################
+# Check file existence
+###############################
+def test_geojson_file_exist():
+    filepath=__location__+"/Testdata/muenster_ring_zeit.geojson"
+    assert os.path.isfile(filepath) == True
+
+def test_nc_file_exist():
+    filepath=__location__+"/Testdata/ECMWF_ERA-40_subset1.nc"
+    assert os.path.isfile(filepath) == True
+
+def test_kml_file_exist():
+    filepath=__location__+"/Testdata/aasee.kml"
+    assert os.path.isfile(filepath) == True
+
+def test_tif_file_exist():
+    filepath=__location__+"/Testdata/wf_100m_klas.tif"
+    assert os.path.isfile(filepath) == True
+
+def test_gpkg_file_exist():
+    filepath=__location__+"/Testdata/nc.gpkg"
+    assert os.path.isfile(filepath) == True
+
+def test_csv_file_exist():
+    filepath=__location__+"/Testdata/cities_NL.csv"
+    assert os.path.isfile(filepath) == True
+
+def test_gml_file_exist():
+    filepath=__location__+"/Testdata/clc_1000_PT.gml"
+    assert os.path.isfile(filepath) == True
+
+def test_shp_file_exist():
+    filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.shp"
+    assert os.path.isfile(filepath) == True
+
+def test_dbf_file_exist():
+    filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.dbf"
+    assert os.path.isfile(filepath) == True
+
+###############################
+# Check bbox extraction
+###############################
+def test_geojson_extract_bbox():
+    filepath=__location__+"/Testdata/muenster_ring_zeit.geojson"
+    geojson = pygeoj.load(filepath)
+    bbox = (geojson).bbox
+    assert bbox == [7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]
+
+def test_invalid_coordinate_geojson_extract_bbox(capsys):
+    filepath=__location__+"/Testdata/invalid_coordinate.geojson"
+    sys.stderr.write("error message")
+    print("Invalid geojson file")
+    out, error = capsys.readouterr()
+    assert out == "Invalid geojson file\n"
+    assert error == "error message"
+
+def test_one_point_geojson_extract_bbox():
+    filepath=__location__+"/Testdata/onePoint.geojson"
+    geojson = pygeoj.load(filepath)
+    bbox = (geojson).bbox
+    assert bbox == [6.220493316650391, 50.52150360276628, 6.220493316650391, 50.52150360276628]
+
+def test_empty_file_geojson_extract_bbox(capsys):
+    filepath=__location__+"/Testdata/empty.geojson"
+    captured = capsys.readouterr()
+    assert captured.out == "Empty file"
+
+def test_nc_extract_bbox():
+    filepath=__location__+"/Testdata/ECMWF_ERA-40_subset1.nc"
+    assert netcdf_bbox(filepath) == [0.0, -90.0, 357.5, 90.0]
+
+def test_shapefile_extract_bbox():
+    filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.shp"
+    sf = shapefile.Reader(filepath)
+    output = sf.bbox
+    print(output)
+    assert output == [295896.274870878, 5694747.64703736, 325999.79578122497, 5747140.98659967]
+
+###############################
+# Functions
```

Comment:
Functions do not belong in the test class but in the library. Please remove here.

Reply:
Done.